README.md

```diff
@@ -230,6 +230,8 @@ export interface SpatialPosition {
 The table schematics can be used for table generation. This is achieved by using the table provided in
 the [angular material package](https://v17.material.angular.io/components/table/overview).
 
+**_NOTE:_** Operations and Events are not included in the code generation by the JS SDK. Generating UI code that represents the Operations that can be performed by an aspect or the events that can be triggered on an aspect has to be manually created. Feel free to contribute / raise requests at https://github.com/eclipse-esmf/esmf-sdk-js-schematics/issues . See https://eclipse-esmf.github.io/ame-guide/modeling/operation.html and https://eclipse-esmf.github.io/ame-guide/modeling/event.html for more information.
```

Comment:
**_NOTE:_** The JavaScript SDK does not automatically generate code for Operations and Events. If you need UI code that represents the Operations available for an aspect, or the Events that an aspect can trigger, you'll need to implement this manually. We welcome contributions and requests for enhancements. Please submit them at our GitHub issues page: [ESMF SDK JS Schematics Issues](https://github.com/eclipse-esmf/esmf-sdk-js-schematics/issues). For additional guidance on modeling Operations and Events, visit our documentation pages on [Modeling Operations](https://eclipse-esmf.github.io/ame-guide/modeling/operation.html) and [Modeling Events](https://eclipse-esmf.github.io/ame-guide/modeling/event.html).